packages/kotti-ui/source/kotti-style/_comments.scss

```diff
@@ -1,127 +1,128 @@
-.comment {
+.kt-comment {
 	display: flex;
 	flex-flow: row;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow
flex-flow is the combo of `flex-direction` and `flex-wrap`
whoever was smart enough to use this, used the default value for flex-direction, and didn't even define the wrap behavior
which means the developer who wrote this basically relied on the two default values

obsolete